.github/workflows/docker.yml

```diff
@@ -89,3 +89,28 @@ jobs:
                   labels: ${{ steps.meta-api.outputs.labels }}
                   cache-from: type=local,src=/tmp/.buildx-cache
                   cache-to: type=local,dest=/tmp/.buildx-cache
+
+            - name: Docker metadata for Clearance UI
```

Comment:
Commit message: Shouldn't "compatible by our deployment" say "compatible *with* our deployment"?

Also, if we want to follow impersonal wording here as well, we should replace "our" with e.g. "Double Open's".